wgpu-core/src/lib.rs

```diff
@@ -211,6 +219,70 @@ pub(crate) fn get_greatest_common_divisor(mut a: u32, mut b: u32) -> u32 {
     }
 }
 
+// XXX TBD CLEANUP NEEDED BELOW - ???
+// XXX TBD SHOULD THIS BE A SEPARATE MODULE - ???
+// XXX TODO CLEANUP WARNING(S) & REMOVE ALLOW(S)
+#[allow(unused_imports)]
+pub(crate) mod alias {
+    pub(crate) use std::prelude::v1::*;
+    pub(crate) mod std {
+        pub(crate) mod prelude {
+            pub(crate) mod v1 {
+                pub(crate) use super::super::{
+                    alloc::format, borrow::ToOwned, boxed::Box, string::String, string::ToString,
+                    vec, vec::Vec,
+                };
+                // XXX TBD ??? - UPDATE FOR std vs test ???
+                // #[cfg(feature = "std")]
+                #[cfg(any(feature = "std", test))]
+                pub(crate) use super::super::{eprintln, thread_local};
+            }
+        }
+        pub(crate) use core::{
+            cell, cmp, convert, ffi, fmt, hash, iter, marker, mem, num, ops, panic, ptr, slice,
+            str, time,
+        };
+        extern crate alloc;
+        pub(crate) use alloc::{boxed, string, vec};
+        pub(crate) mod borrow {
+            pub(crate) use super::alloc::borrow::*;
+        }
+        pub(crate) mod sync {
+            pub(crate) use super::alloc::sync::*;
+            #[cfg(any(feature = "std", test))]
+            pub(crate) use super::std::sync::*;
+            pub(crate) use core::sync::*;
+            // XXX TBD ??? - UPDATE FOR std vs test ???
+            // XXX TBD NAMING - ???
+            // XXX TODO UNCOMMENT - XXX XXX THIS COMMENTED-OUT IMPORT SHOULD TRIGGER CI BUILD FAILURE
+            // #[cfg(not(any(feature = "std", test)))]
+            // pub(crate) use crate::OnceCell as OnceLock;
+        }
+        // XXX TBD ??? - UPDATE FOR std vs test ???
+        // #[cfg(feature = "std")]
+        #[cfg(any(feature = "std", test))]
+        extern crate std;
+        // XXX TBD ??? - UPDATE FOR std vs test ???
+        // #[cfg(feature = "std")]
+        #[cfg(any(feature = "std", test))]
+        pub(crate) use std::{
+            backtrace, env, eprintln, fs, io, path, process, thread, thread_local,
+        };
+        // XXX TBD MOVE THIS - ???
+        // XXX TODO ADD NOTE - IMPORTING error from std, if possible, to avoid MSRV issue
+        // XXX TBD ??? - UPDATE FOR std vs test ???
+        // #[cfg(feature = "std")]
+        #[cfg(any(feature = "std", test))]
+        pub(crate) use std::error;
+        // XXX TODO UNCOMMENT - XXX XXX THIS COMMENTED-OUT IMPORT SHOULD TRIGGER CI BUILD FAILURE
+        // XXX TBD THIS IMPORT (IF UNCOMMENTED) IS EXPECTED TO BREAK MSRV - XXX TODO CI SHOULD DETECT THIS
+        // #[cfg(not(any(feature = "std", test)))]
+        // pub(crate) use core::error;
```

Comment:
CI should have failed with the missing import for no-std build - see https://github.com/gfx-rs/wgpu/pull/6926/files#r1919365322 (I will propose a solution for this)

Reply:
You just pung a random user lol - don't worry about it, we can always change notification settings